optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -263,6 +274,28 @@ export const TrialTable: FC<{
   ]
 
   const collapseBody = (index: number) => {
+    const objectiveFormRefs = studyDetail?.directions.map((d) =>
+      createRef<HTMLInputElement>()
+    )
+    const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
+      if (objectiveFormRefs === undefined) {
+        return
+      }
+
+      e.preventDefault()
+      const studyId = (studyDetail as StudyDetail).id
```

Comment:
You should not cast `StudyDetail | null` to `StudyDetail`. It's not safe. Please return this function if `studyDetail === null`.

Reply:
Fixed